common/src/View/MapFrame.cpp

```diff
@@ -877,6 +879,19 @@ namespace TrenchBroom {
             }
         }
 
+        void MapFrame::OnFileReloadTextureCollections(wxCommandEvent& event) {
+            if (IsBeingDeleted()) return;
+
+            m_document->reloadTextureCollections();
+        }
+
+        void MapFrame::OnFileReloadEntityDefinitions(wxCommandEvent& event) {
+            if (IsBeingDeleted()) return;
+
+            auto oldSpec = m_document->entityDefinitionFile();
```

Comment:
Could you spend a separate method to reload the entity definitions similar to reloadTextureCollections? It seems odd to have it for one case but not the other.